xray/utils/resultstable.go

```diff
@@ -28,23 +31,37 @@ const (
 // In case multipleRoots is true, the field Component will show the root of each impact path, otherwise it will show the root's child.
 // In case one (or more) of the violations contains the field FailBuild set to true, CliError with exit code 3 will be returned.
 // Set printExtended to true to print fields with 'extended' tag.
-func PrintViolationsTable(violations []services.Violation, multipleRoots, printExtended bool) error {
+func PrintViolationsTable(violations []services.Violation, multipleRoots, printExtended, isScan bool) error {
```

Comment:
The below a[[ly also for the `PrintVulnerabilitiesTable`, `PrintScanResults` and `PrintLicensesTable` functions - 

1. isScan --> scan
2. Let's describe the arguments this function receieves, and specifically the new `scan` argument.